rakelib/artifacts.rake

```diff
@@ -669,7 +669,7 @@ namespace "artifact" do
       elsif stat.symlink?
         tar.symlink(path_in_tar, File.readlink(path), :mode => stat.mode)
       else
-        tar.add_file_simple(path_in_tar, :mode => stat.mode, :size => stat.size) do |io|
+        tar.add_file_simple(path_in_tar, :mode => stat.mode, :size => stat.size, :mtime => stat.mtime&.to_i || Time.now.to_i) do |io|
```

Comment:
to be fair, with & we'll never get nil on the left hand side, so the || is never invoked. I'd be tempted just to have `stat.mtime.to_i` without and OR, but the silent nil => epoch worries me..let's do instead:

```suggestion
        tar.add_file_simple(path_in_tar, :mode => stat.mode, :size => stat.size, :mtime => (stat.mtime || Time.now).to_i) do |io|
```

Reply:
Good point. I'll commit this and merge. I want to double check the artifacts produced in CI have timestamps that make sense!